jestConfig.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  collectCoverage: true,
```

Comment:
Is coverage generated for this project? Do we need coverage for jsx?

Reply:
Coverage is generated. We can simplify the coverage to just `*.js` for sure.

Reply:
This has landed in master: https://github.com/cerner/terra-enzyme-intl/blob/master/jestConfig.js